File: media-service/src/main/java/com/mordent/ua/mediaservice/repository/SongMetadataRepository.java
package com.mordent.ua.mediaservice.repository;

import com.mordent.ua.mediaservice.model.data.SongMetadata;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SongMetadataRepository extends ReactiveMongoRepository<SongMetadata, Long> {
}

File: metadata-service/src/main/kotlin/com/mordent/ua/metadataservice/repository/SongMetadataRepository.kt
package com.mordent.ua.metadataservice.repository

import com.mordent.ua.metadataservice.model.data.SongMetadata
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository

@Repository
interface SongMetadataRepository: ReactiveMongoRepository<SongMetadata, Long> {
}